script/seed.js

```diff
@@ -22,35 +22,59 @@ async function seed() {
       firstName: 'Murphy',
       lastName: 'Clark'
     })
-  ])
+  ]);
 
-  console.log(`seeded ${users.length} users`)
-  console.log(`seeded successfully`)
+  const products = await Promise.all([
+    Product.create({
+      name: 'Sloth Bed'
+    }),
+    Product.create({
+      name: 'Sloth Mask'
+    })
+  ]);
+
+  const carts = await Promise.all([
+    Cart.create({
+      userId: 1,
```

Comment:
the userId and productId in carts need to be dynamically generated from your users and products array (e.g. users[0].id and products[1].id) instead of 0 and 1, because you can't predict the ids beforehand

Reply:
Added two methods to the Cart model that gets the Product ID and User Id from those respective models.